scripts/issue/1338/deprecate_avatars.py

```diff
@@ -0,0 +1,70 @@
+from brownie import web3
+from decimal import Decimal
+from great_ape_safe import GreatApeSafe
+from helpers.addresses import r
+
+
+# https://etherscan.io/address/0x5a92EF27f4baA7C766aee6d751f754EBdEBd9fae#code#L722
+MIN_LOCK_TIME = 594000
+
+
+def main():
+    vault = GreatApeSafe(r.badger_wallets.treasury_vault_multisig)
+    vault.init_convex()
+    vault.init_aura()
+
+    # avatars
+    aura_avatar = vault.contract(r.avatars.aura)
+    convex_avatar = vault.contract(r.avatars.convex)
+
+    # balancer bpts
+    bpt_badgerwbtc = vault.contract(r.balancer.B_20_BTC_80_BADGER)
+    bpt_badgerreth = vault.contract(r.balancer.B_50_BADGER_50_RETH)
+    bpt_wbtcdigggravi = vault.contract(r.balancer.bpt_40wbtc_40digg_20graviaura)
+
+    # convex wrap lp
+    badger_fraxbp = vault.contract(r.treasury_tokens.badgerFRAXBP_f_lp)
+    wcvx_badger_fraxbp = vault.contract(r.convex.frax.wcvx_badger_fraxbp)
+
+    # liq out from avatars
+    aura_avatar.withdrawAll()
+
+    private_pids = convex_avatar.getPrivateVaultPids()
+    if len(private_pids) > 0:
+        for pid in private_pids:
+            convex_avatar.withdrawFromPrivateVault(pid)
+
+    # deposit in vault directly pos
```

Comment:
If possible, it might make sense to add a balance check here for all the above withdrawals to ensure no liquidity have been left behind

Reply:
Agree here, there's shouldn't be any balance changes outside of the operation so I think it would even be safe to do the balance check on-chain with the `balance_checker`